sycl/doc/extensions/fpga_io_pipes_design.rst

```diff
@@ -22,7 +22,7 @@ Requirements
 
 Links
 -----
-.. _Spec: https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/DataFlowPipes/data_flow_pipes.asciidoc
+.. _Spec: https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/SYCL_EXT_INTEL_DATAFLOW_PIPES.asciidoc
```

Comment:
Not sure how to make relative link here :(

Reply:
I also don't know reStructuredText that well.  When I look at the HTML rendering of this file in GitHub, all the content in this area of the document is missing.  It seems like the GitHub renderer doesn't understand the `.. code` directive. So, there are bigger problems than the absolute vs. relative URL link.